04-Multi-Band-Raster.Rmd

```diff
@@ -70,22 +65,36 @@ Read more about the `raster` package in R.</a>
 
 #About Raster Bands
 
-As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}/R/Introduction-to-Raster-Data-In-R), a raster can contain 1 or more bands. To 
-work with multi-band rasters, we need to adjust how we import and plot our data. 
+As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), a raster can
+contain 1 or more bands. To work with multi-band rasters, we need to change how 
+we import and plot our data in several ways. 
 
-* To import multi band raster data we will use the `stack` function.
-* If our multi-band data are imagery that we wish to composit, we can use `plotRGB` 
-to plot a 3 band image raster.
+* To import multi band raster data we will use the `stack()` function.
+* If our multi-band data are imagery that we wish to composite (spatial data
+term essentially meaning, to view multiple bands together), we can use
+`plotRGB()` to plot a 3 band image raster.
+
+>Bonus: Bands? Layers?  Why are both names used? The term bands comes from the 
```

Comment:
Sounds good. Honestly i'd REMOVE the bonus and just say something like
Want to know more about bands in remote sensing imagery, watch this youtube video.

We can embed the video in the lesson! Code below.

<iframe width="560" height="315" src="https://www.youtube.com/embed/3iaFzafWJQE" frameborder="0" allowfullscreen></iframe>
Leah
--
Leah A. Wasser, Ph.D.
Supervising Scientist
Education / Data Products
National Ecological Observatory Network (NEON)
Boulder, Colorado
--------------------
For free tutorials and resources on working with spatio-temporal data:
http://www.neondataskills.org<http://www.neondataskills.org/>

From: "Megan A. Jones" <notifications@github.com<mailto:notifications@github.com>>
Reply-To: data-lessons/NEON-R-Spatial-Raster <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, November 16, 2015 at 2:09 PM
To: data-lessons/NEON-R-Spatial-Raster <NEON-R-Spatial-Raster@noreply.github.com<mailto:NEON-R-Spatial-Raster@noreply.github.com>>
Cc: Leah Wasser <lwasser@neoninc.org<mailto:lwasser@neoninc.org>>
Subject: Re: [NEON-R-Spatial-Raster] Lessons 03 and 04 Rearrangements and Edits (#27)


In 04-Multi-Band-Raster.Rmd<https://github.com/data-lessons/NEON-R-Spatial-Raster/pull/27#discussion_r44983469>:

>
> -* To import multi band raster data we will use the `stack` function.
> -* If our multi-band data are imagery that we wish to composit, we can use `plotRGB`
> -to plot a 3 band image raster.
> +* To import multi band raster data we will use the `stack()` function.
> +* If our multi-band data are imagery that we wish to composite (spatial data
> +term essentially meaning, to view multiple bands together), we can use
> +`plotRGB()` to plot a 3 band image raster.
> +
> +>Bonus: Bands? Layers?  Why are both names used? The term bands comes from the


How about 1)Change all layers to bands except were we really mean layer. 2) Substantially shorten this bonus. 3) Add in the Mapping The Invisible video to the bonus bit: https://www.youtube.com/watch?v=3iaFzafWJQE&noredirect=1.

-
Reply to this email directly or view it on GitHub<https://github.com/data-lessons/NEON-R-Spatial-Raster/pull/27/files#r44983469>.